src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Jobs2dAdapter.java

```diff
@@ -27,11 +28,14 @@ public void operateTo(int x, int y) {
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
 
-		drawLine(line);
+		this.drawLine(line);
+		this.setPosition(x, y);
+
+		DrawerFeature.getDrawerController().drawLine(line);
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.